CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Refinery Changelog
 
+## 1.20.0 2023-03-10
+
+### Summary
+This is a significant new release of Refinery, with several features designed to help when operating Refinery at scale:
+
+- Changes the cache model for recording the history of kept and dropped traces; changing the SampleCacheStrategy `Type` to `cuckoo` will enable trace decisions to be kept for much longer durations. See the [Sample Cache Strategy](https://github.com/honeycombio/refinery/blob/main/config_complete.toml#L447) section of config_complete.toml for more.
+- When refinery shuts down, it will try to remove itself from the peers list, which should shorten the time of instability in the cluster.
+- The algorithm controlling how traces are distributed to nodes in the cluster has been revamped so that traces are much more likely to stay on the same node during reconfiguration. Set the [Peer Management](https://github.com/honeycombio/refinery/blob/main/config_complete.toml#L269) `Strategy` to `hash` to enable it.
+- A [Stress Relief](https://github.com/honeycombio/refinery/blob/main/config_complete.toml#L493) system has been added. When properly configured, it tracks refinery's load, and if it gets in danger of instability, switches into a high-performance mode designed to relieve stress on the system. When Activated, refinery samples spans at a deterministic rate and will continue doing so until the load is reduced. It also indicates which of its configuration values is most under stress, which should help tune it.
```

Comment:
I recommend mentioning the metric to monitor for this and its default value for when Stress Relief is activates.